migrator/migration.py

```diff
@@ -402,3 +421,9 @@ def parse_cloudfront_error_response(error_responses):
         for item in error_responses.get("Items", []):
             responses[item["ResponseCode"]] = item["ResponsePagePath"]
         return responses
+
+
+class DomainMigration(Migration):
+    def __init__(self, route, session, client):
+        self.domains = route.domains
+        super().__init__(route, session, client)
```

Comment:
Just checking that `DomainMigration` does not need the same attributes as the `CdnMigration` class, is that correct?

Reply:
it doesn't need all of them - the broker's `ALBServiceInstance`s are way simpler than `CDNServiceInstance`s, so the `DomainMigration` is way simpler than the `CdnMigration`